Resources/Prototypes/Nyanotrasen/Entities/Objects/Devices/CircuitBoards/production.yml

```diff
@@ -44,3 +44,22 @@
     - type: ReverseEngineering
       recipes:
         - DeepFryerMachineCircuitboard
+
+- type: entity
+  id: MailTeleporterMachineCircuitboard
+  parent: BaseMachineCircuitboard
+  name: mail teleporter machine board
+  components:
+    - type: Sprite
+      state: supply
+    - type: MachineBoard
+      prototype: MailTeleporter
+      requirements:
+        Capacitor: 1
+        MatterBin: 1
+      materialRequirements:
+        Steel: 4
+        Cable: 4
+    - type: ReverseEngineering
```

Comment:
i can already see this being reverse engineered to make 20 mail teleporters for either epic money or to drain cargos bank account automatically

also it has no recipe anyway

Reply:
You are right on that one, should probably make it non reverser-able. Even tho, yeah, no recipe. Might aswell just make sure.